packages/frontend/src/translations/en.global.json

```diff
@@ -1411,6 +1411,7 @@
         "desc": "Protect your account by requiring confirmation via SMS or email when authorizing transactions.",
         "disable": {
             "desc": "Keep in mind that transactions won't have to be confirmed with 2FA once it's disabled.",
+            "phoneDesc": "The SMS option for two-factor authentication is being deprecated, and cannot be re-enbaled.",
```

Comment:
How to go about updating other languages?

Reply:
The UI will display the string in English while there's no corresponding translation available, which I think is reasonable here since we want to prioritize shipping this change